src/components/Header/Header.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext, useState } from 'react';
+import { creatTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+import { AuthContext } from '../Auth/AuthContext';
+
+type Props = {
+  submitTodo: (todo: Todo) => void,
+  onSetError: (message: string) => void,
+  onAddTempTodo: (title: string) => void,
+  onSetTempTodo: (tempTodo: null) => void,
+  onSetLoading: (isLoading: boolean) => void,
+};
+
+export const Header: React.FC<Props> = ({
+  submitTodo,
+  onSetError,
+  onAddTempTodo,
+  onSetTempTodo,
+  onSetLoading,
+}) => {
+  const [title, setTitle] = useState('');
+  const [isDisabled, setIsDisabled] = useState(false);
```

Comment:
isWhatDisabled?

Reply:
@naziamoff The input  during loading data